flutter_secure_storage_web/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: flutter_secure_storage_web
 description: Web implementation of flutter_secure_storage. Use flutter_secure_storage for the full flutter package.
 repository: https://github.com/mogol/flutter_secure_storage
-version: 1.1.2
+version: 1.2.0
 
 environment:
   sdk: ">=2.12.0 <4.0.0"
```

Comment:
Everything looks good, however the only thing that's a bit annoying is that the web package is available from dart 3.3, which is basically the latest flutter version, meaning that if we publish this, everyone is required to run the latest Flutter version.

Reply:
Yes, this is true. I have pushed the constraints for `web` down as far as possible. Older versions lack features that are (IMHO) critical for the functionality of `flutter_secure_storage`. Maybe, the constraints *could* be pushed even lower than that using some hacky workarounds, but I am unsure.
This is also one of the reasons why it makes sense to apply this PR (and #698 for that matter) only after a major version bump